README.md

```diff
@@ -7,194 +7,107 @@
 
 This repository contains code to build ManageIQ appliances in the various virtualization formats.
 
-It depends on the following git repositories:
-
-* [ManageIQ](https://github.com/ManageIQ/manageiq) - ManageIQ application code
-* [ManageIQ-Appliance](https://github.com/ManageIQ/manageiq-appliance) - System files for ManageIQ appliances
-
-Below are instructions on installing and configuring a virtual machine to generate appliance images.
+Below are instructions on configuring a machine to generate appliance images.
 
 # Installation
 
   **Note: All instructions are as root unless specified**
 
-## Setup CentOS 7.2 in a new virtual machine
+## Setup CentOS 8
+
+  * RAM: 12GB minimum
+  * HD: 80GB Minimum - 200GB Recommended
 
-  * CentOS-7-x86_64-DVD-1511-01.iso (attached to vm)
-  * Vm configuration:
-    * 8GB Ram
-    * HD 80GB Minimum - 200GB Recommended
+  * If setting up as a VM:
```

Comment:
"If setting up the build machine as a VM"

Reply:
Done